### DISCUSSION TITLE
AttributeError: 'bool' object has no attribute 'sum'

### DISCUSSION THREAD

**Discussion by PhD-GOAT** [raphaelvallat/yasa#62] [Q&A] [answered]
[example notebook](https://github.com/raphaelvallat/yasa/blob/master/notebooks/14_automatic_sleep_staging.ipynb)

---> 24 accuracy = (hypno == y_pred).sum() / y_pred.size

In this case, it was edf, not fif! I wonder if you can't do it with an edf.

**Comment by raphaelvallat** [comment, marked as answer]
Yes you can use the mne.io.read_raw_edf to load an EDF file, and then all the subsequent steps should be the same.

I think that there is something wrong with your `hypno` variable, it should be a vector and not a boolean (True / False).

**Reply by PhD-GOAT** [reply]
@raphaelvallat 

The dtype of hypno is str. The size of hypno is more y_pred. Is this the problem? It says that it is an error of the bool type, but I don't know how to do it with a vector.
